src/shared/ui/survey/MediaItems/MediaInput.tsx

```diff
@@ -44,9 +55,13 @@ const MediaInput: FC<Props> = ({
   };
 
   return (
-    <TouchableOpacity onPress={onUploadPress}>
-      <UploadButton>{children}</UploadButton>
-    </TouchableOpacity>
+    <UploadButton>
```

Comment:
It seems UploadButton is just an area where internals placed. This not a button. Please rename to something like "ContentWrapper"